packages/protobuf/src/to-binary.ts

```diff
@@ -80,7 +80,7 @@ function writeFields(
   }
   if (opts.writeUnknownFields) {
     for (const { no, wireType, data } of msg.getUnknown() ?? []) {
-      writer.tag(no, wireType).raw(data);
+      writer.tag(no, wireType).bytes(data);
```

Comment:
The `bytes` method prefixes the length. The change will corrupt data, and needs to be reverted.